cmd/kill/kill.go

```diff
@@ -0,0 +1,59 @@
+package main
+
+import (
+	"os"
+	"strconv"
+	"fmt"
+	"flag"
+
+	killer "github.com/tiago4orion/enzo/cmd/kill/killer"
+)
+
+// save own name for easier reference
+var me = os.Args[0]
+
+func usage() {
+	fmt.Println("Usage:")
+	fmt.Println("kill [-safe] pids")
+	flag.PrintDefaults()
+	os.Exit(22); // EINVAL
```

Comment:
I agree. Mainly considering that `kill` can be used on windows too, so error codes would make no sense. What do you think of create package with constant errors to be used in entire project?